packages/core/upload/admin/src/translations/en.json

```diff
@@ -14,7 +14,7 @@
   "control-card.stop-crop": "Stop cropping",
   "filter.add": "Add filter",
   "form.button.replace-media": "Replace media",
-  "form.input.decription.file-alt": "This text will be displayed if the asset can’t be shown.",
+  "form.input.description.file-alt": "This text will be displayed if the asset can’t be shown.",
```

Comment:
I generally don't touch non English localizations -- assuming that translators will deal w/ changes. But, I can if requested...

Reply:
If you update a translation key you need to update it for all of them too. Thank you